### ISSUE TITLE
tube8.com ads [NSFW]

### ISSUE THREAD

**Issue by xDarkARG** [uBlockOrigin/uAssets#900]
URL(s) where the issue occurs

https://www.tube8.com/anal/raunchy-anal-session-with-a-stunning-blonde/36511511/

Describe the issue

ADS

Versions

Browser/version: Chrome 62.0.3202.62
uBlock Origin version: 1.14.20

Settings

I use the configuration that comes already in ublock when one installs the Extension

**Comment by in4u** [comment]
There's an ad banner now above the video.

**Comment by mapx-** [comment]
the thumbnails when you pause the video ?
`tube8.com##[class$="thumbnailsGrid"]`

**Comment by in4u** [comment]
I was talking about another ad. It was similar to one I pointed out for biqle.ru and perhaps a few others recently. I don't see it now. I think Easylist updated to blanket remove all such ads with `##a[href^="https://www.brazzersnetwork.com/landing/"]` rule which took care of it.

But there's an updated ad now going around which should be blocked with `##a[href^="https://www.brazzersnetwork.com/landing2/"]`.

Also, the thumbnail ads are new. Weren't there at the time of my commenting. I think [Mindgeek](https://en.wikipedia.org/wiki/MindGeek) is rolling out overlay ad on video pause on all their network sites slowly. I opened issues for youporn and redtube earlier and now tube8 got it too.
